lennykamande/django_api
# Ft tdd models 179077854

#### What does this PR do?
Have TDD running so that we can now build the app and test before writing code
#### Description of Task to be completed?
Setup of the base application that should take in points and database setup on postgres
#### How should this be manually tested?
After clonning the repo one should simply install docker and cd into the django_api folder

- run the command "sudo docker-compose up --build" 

#### Any background context you want to provide?
Setup of a basic api to calculate points 
#### What are the relevant pivotal tracker stories?
179077854
#### Screenshots (if appropriate)
#### Questions
